### ISSUE TITLE
weighting add support for min capping

### ISSUE THREAD

**Issue by jamesrkg** [Quantipy/quantipy#617]
We want the ability to set min and max capping in a weight scheme, e.g.:

```python
scheme = qp.Rim("My weight scheme", cap=(0.05, 5))
```

Which would make sure no factors were less than `0.05` or more than `5`.

Existing use will be interpreted as max-only capping, e.g.:

```python
scheme = qp.Rim("My weight scheme", cap=5)
```

**Comment by jamesrkg** [comment]
Opt out of anesrake cap correction using: 

```python
scheme.anesrake_cap_correction = False
```

**Comment by jamesrkg** [comment]
This PR resolved by #618.
